### PR TITLE
Code Refactoring and Testing

### PR DESCRIPTION
**Pull request by Zeus3101**
Expanded all vague variable names and retested code.

Found a few issues:

- VPG and PPO1 are missing get_policy_from_name and get_value_from_name import statements
- Bandits.py is not running because it's getting invalid (probably zero) values as arguments for np.sqrt() and np.log()
- VPG had a redundant evaluate method (removed)
- algo.load() function was used incorrectly (takes only one argument but two were given)

### CODE REVIEW

File: jigglypuffRL/common/base.py [hunk 1]
```diff
@@ -14,25 +14,25 @@ def __init__(self, disc, det, **kwargs):
...
+        return self.model(state)
 
-    def get_action(self, s):
-        _s = self.forward(s)
+    def get_action(self, state):
+        ps = self.forward(state)
 
         if self.disc:
-            _s = nn.Softmax(dim=-1)(_s)
+            ps = nn.Softmax(dim=-1)(ps)
```
Comment (Sharad24) [inline jigglypuffRL/common/base.py:24]: Can you change `ps` and `disc` as well here?

File: jigglypuffRL/deeprl/ddpg/ddpg.py [hunk 2]
```diff
@@ -163,104 +163,113 @@ def create_model(self):
...
+        q = self.ac.critic.get_value(torch.cat([state, action], dim=-1))
 
         with torch.no_grad():
-            # print(s1.shape, self.ac_targ.get_action(s1).shape)
-            q_pi_targ = self.ac_targ.get_value(
-                torch.cat([s1, self.ac_targ.get_action(s1)], dim=-1)
-            )
-            target = r + self.gamma * (1 - d) * q_pi_targ
+            q_pi_targ = self.ac_targ.get_value(torch.cat([
+                next_state, self.ac_targ.get_action(next_state)
```
Comment (Sharad24) [inline jigglypuffRL/deeprl/ddpg/ddpg.py:187]: `targ`

File: jigglypuffRL/deeprl/ppo1/ppo1.py [hunk 3]
```diff
@@ -172,8 +172,8 @@ def get_traj_loss(self):
         returns = []
 
         # calculate discounted return
-        for r in self.policy_old.traj_reward[::-1]:
-            R = r + self.gamma * R
+        for reward in self.policy_old.traj_reward[::-1]:
+            R = reward + self.gamma * R
```
Comment (Sharad24) [inline jigglypuffRL/deeprl/ppo1/ppo1.py:176]: `R`

**Comment by Zeus3101** [comment]
Do take note of the second issue. I've solved the fourth issue in another PR #63 . The other two are already resolved.

**Pull request merged**